### ISSUE TITLE
a11y: Fix semantically incorrect tables

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13818]
a11y issue 86

**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Table headers not programmatically defined

The text present inside the table are not marked-up as row headers resulting  in screen reader users finding it difficult to understand the table structure effectively.

Account and API Keys
Account Section
- Name:
- Username:
- Type:

About
Image list section
-Linux
- Windows
Cli Download section
-  Mac
- Linux
- Windows 

Use `<th>` tags to identify row headers such as "Name:", "Username:" and "Type:" of the table.

**To Reproduce**

**Result**

**Expected Result**

The headers should be identified for the associated data of the table by screen reader for their users.

**Screenshots**

**Additional context**

**Comment by a-arias** [comment]
TESTS
Rancher : v2.12-44000c76dcb2a50d5dd4917555a5a25967aa3809-head
UI: master https://github.com/rancher/dashboard/commit/22633ef3eb8248aa7916f318b536b22f3fbc2e12

Fix validated, we added Th on the next sections:
Account and API Keys
Account Section
Name:
Username:
Type:

About
Image list section
-Linux

Windows
Cli Download section
Mac
Linux
Windows

<img width="1180" alt="Image" src="https://github.com/user-attachments/assets/d5a4bcc6-afb5-445c-a8e7-9b76ab03ffca" />
<img width="1180" alt="Image" src="https://github.com/user-attachments/assets/a46fc264-cc03-4c61-9c80-2792e8cccaa2" />
